conda-reqs/firesim.yaml

```diff
@@ -30,13 +30,13 @@ dependencies:
     - binutils
 
     - dromajo # from ucb-bar channel - https://github.com/riscv-boom/dromajo
-    - riscv-tools # from ucb-bar channel - https://github.com/ucb-bar/riscv-tools-feedstock
+    - riscv-tools=1.0.1 # from ucb-bar channel - https://github.com/ucb-bar/riscv-tools-feedstock
 
     # firemarshal deps
     - qemu # from ucb-bar channel - https://github.com/ucb-bar/qemu-feedstock
     - rsync
     - psutil
-    - doit=0.35.0
+    - doit>=0.35.0
```

Comment:
I thought this This was added with `=`, referencing a firemarsal issue in the commit message.  However, I went back through things and it looks like Abe added it in https://github.com/firesim/firesim/commit/aed821b504607ec8cc33187de1a892f2fee9c4b7 without really pointing to why.  The firemarshal reqs say `>0.34`

Reply:
Good catch. This can be `>=0.34` based on this: https://github.com/firesim/FireMarshal/pull/242.